### PR TITLE
Add legal information pages

### PR DESCRIPTION
**Pull request by Aumarh**
- [x] create legal pages
 - [x]  route to legal pages from sign up page

### CODE REVIEW

File: frontend/pages/legal/acceptable-use-policy.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    <meta name="description" content="acceptable use policy" />
+    <meta name="author" content="" />
+
+    <!-- Le styles -->
+    <link href="/css/bootstrap.css" rel="stylesheet" />
+    <link href="/css/bootstrap-responsive.css" rel="stylesheet" />
+    <link href="/css/style.css" rel="stylesheet" />
+    <link href="/css/datepicker.css" rel="stylesheet" />
+    <link href="/css/jquery-ui-1.8.16.custom.css" rel="stylesheet" />
+    <link href="/css/jquery-ui-timepicker-addon.css" rel="stylesheet" />
```
Comment (dimakrsna) [inline frontend/pages/legal/acceptable-use-policy.html:17]: React applications do not use jquery plugins

**Review by dimakrsna — CHANGES_REQUESTED** [review]
@Aumarh All new pages must have a .jsx extension and be React components. Styles must be connected by analogy with other pages (from the styles folder)

File: frontend/components/SignUpForm/index.tsx [hunk 2]
```diff
@@ -1,37 +1,41 @@
...
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { FormEvent, useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+import { useLazySignUpQuery } from '../../store/api/authApi';
+import { ApiTypes } from '../../types';
+import { signUpSchema, validationInitialState } from '../../utils/validations';
+import { FormInput } from '../FormInput';
+import { Spiner } from '../Spiner';
+import styles from './index.module.scss';
```
[the remark this answers is not in the document]
Comment (Aumarh) [reply frontend/components/SignUpForm/index.tsx]: > 

Thanks Dmytro, did that before this push.

File: frontend/pages/legal/acceptable-use-policy.html [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Acceptable Use Policy</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="acceptable use policy" />
+    <meta name="author" content="" />
+
+    <!-- Le styles -->
+    <link href="/css/bootstrap.css" rel="stylesheet" />
```
[the remark this answers is not in the document]
Comment (Aumarh) [reply frontend/pages/legal/acceptable-use-policy.html]: removed

File: frontend/pages/legal/acceptable-use-policy.html (hunk 1 above)
Comment (Aumarh) [reply frontend/pages/legal/acceptable-use-policy.html]: removed

**Comment by Aumarh** [comment]
> @Aumarh All new pages must have a .jsx extension and be React components. Styles must be connected by analogy with other pages (from the styles folder)

Done

**Review by dimakrsna — APPROVED** [review]
@Aumarh Thanks for the updates! Please take a look on my last commits, I am was adding some improvements.

**Pull request merged**